otterdog/eclipse-score.jsonnet

```
@@ -809,7 +812,7 @@ orgs.newOrg('automotive.score', 'eclipse-score') {
       ],
       description: "Incubation repo for orchestration",
```

Description has to be adjusted, this is not an incubator anymore. + we can hint to the async executor within the orchestrator
```suggestion
      description: "Repository for the Orchestration framework incl. async executor",
```